### PR TITLE
Clippy fixes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -20,3 +20,5 @@ jobs:
       run: cargo build --verbose
     - name: Run tests
       run: cargo test --verbose
+    - name: Clippy
```
Comment (asiniscalchi) [inline .github/workflows/rust.yml]: Is it logical to perform linting before initiating the build process? This suggestion stems from the fact that the build process could be time-consuming, and discovering an error after such a long wait could be quite vexing.

Comment (ghost) [reply .github/workflows/rust.yml]: fixed it

**Review by asiniscalchi — APPROVED** [review]

File: src/benchmarking.rs [hunk 2]
```diff
@@ -3,18 +3,18 @@
 use super::*;
 
 #[allow(unused)]
-use crate::Pallet as Template;
+use crate::Pallet as LivingAssetsOwnership;
```
Comment (tonimateos) [inline src/benchmarking.rs:6]: Are these changes related to the Clippy linting at all?

**Review by tonimateos — COMMENTED** [review]
Looks like one change is not related to the name/goal of the PR, can it be?

File: src/benchmarking.rs (hunk 2 above)
Comment (ghost) [reply src/benchmarking.rs:6]: oh yeah, I guess you are right here. I saw a lot of clippy errors in the `benchmarking` files and I thought I could also just refactor them while I am here
